status-im/status-react
# [ISSUE #3537] Added mixpanel support

fixes #3537
fixes #3545

### Summary:

Added support for mixpanel. Some actions triggers mixpanel events

### Steps to test Mixpanel:
- Open Status
- Log in
- Opt-in for stats
- Verify events are logged in mixpanel

### Steps to test Opt-out:
- Open Status
- Log in
- Opt-OUT for stats
- Verify no events are sent to Mixpanel at all

status: ready
